app/controllers/apartments_controller.rb

```diff
@@ -3,4 +3,8 @@ def index
     apartments = Apartment.all
     render json:apartments
   end
+  def create
+    apartment = Apartment.create(name: params[:name], age: params[:age], enjoys: params[:enjoys])
```

Comment:
You need strong params here. And I think some cat tinder code snuck in here.

Reply:
Okay thanks! I should have commented that out since I am just now starting my create functionality.

Reply:
pushed changes